apps/console/src/extensions/configs/analytics.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Lets update all licenses to the new version. Update other instances as well.

Reply:
Fixed in #2729